Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,22 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//  Answer : there are 6 variable declarations
 
 // b) How many function calls are there?
+// Answer: zero function call
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// Answer: % is a modulus operator, it returns integer remainder of 2 operands , here 8784 % 60 is 24 
+//  used to calculate the remaining seconds (24 sec)
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+// Answer: it means total movie length in minutes
```

Comment:
if `movieLength` is 59, according to your description, we would say the movie length is 0 minutes. 
I wonder if there is a better way to describe it. I wonder how ChatGPT would describe it.

Reply:
I used _ChatGPT to explore_ how best to describe short movie lengths. If the movie length is 59 seconds, instead of saying "0 minutes," it might be clearer to say “under 1 minute” or “59 seconds” directly. This avoids any confusion about the duration, accurately reflecting the brief length without implying zero time. So, it suggested to use if , else block to display result in correct format 
**For example :**
 If movieLength is 59, the output would be "59s".
For times over a minute but under an hour, like 123 seconds, it would display "2m 3s".
For times over an hour, it would display all parts, such as "2h 26m 24s" for 8784 seconds.